.github/workflows/test-plugins.yml

```diff
@@ -18,6 +18,13 @@ permissions:
 jobs:
   run-scripts:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        range:
+          - { from: "0", to: "b" }
+          - { from: "c", to: "n" }
+          - { from: "o", to: "z" }
```

Comment:
There's a theme with the name `_tk`, so we will need to ensure special characters are supported. https://wordpress.org/themes/_tk/

I'm not sure which characters are supported, `_` is the only one I found.